dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/page/PageResourceTest.java

```diff
@@ -553,4 +555,54 @@ public void testRenderPersonalizationVersion() throws DotDataException, DotSecur
 
         assertNull(pageView.getViewAs().getPersona());
     }
+
+    /***
+     * Should render containers
+     *
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @Test
+    public void testShoudrendercContainers() throws DotDataException, DotSecurityException, InterruptedException {
```

Comment:
I think the test should be renamed to `testShouldRenderContainers`

Reply:
Actually we are missing things in the name. Please give a better name following the `methodToTest_GivenScenario_ExpectedResult` or fill a better description in the javadoc section